Java/ds/.deepsource.toml

```diff
@@ -0,0 +1,19 @@
+version = 1
+
+[[analyzers]]
+name = "python"
+enabled = true
+
+  [analyzers.meta]
+  runtime_version = "3.x.x"
+
+[[analyzers]]
+name = "go"
+enabled = true
+
+  [analyzers.meta]
+  import_paths = ["github.com/Jassi10000/NeoAlgo"]
+
+[[analyzers]]
+name = "javascript"
+enabled = true
```

Comment:
Why have you put this here?

Reply:
Actually It was giving deepsource errors ,so I thought of adding this
No issues , I will remove it 
and send a correct PR